app/app.model.js

```diff
@@ -32,11 +32,26 @@ exports.selectArticleById = (article_id) => {
   });
 };
 
-exports.selectAllArticles = () => {
+exports.selectAllArticles = (topic) => {
+  const queryValues = [];
+  const validTopics = ["cats", "mitch", "coding", "football", "cooking"];
```

Comment:
the topics can't be hard coded.  this is test data!   So when you come to use the production data, non of these topics will not align and you'll get a 404.    You need check if the topic exists the db.